include/secp256k1.h

```diff
@@ -80,6 +80,26 @@ typedef struct {
     unsigned char data[64];
 } secp256k1_ecdsa_signature;
 
+/** Data structure that holds a sign-to-contract ("s2c") context. Sign-to-contract
+ *  allows a signer to commit to some data as part of a signature. If the nonce
+ *  function supports sign-to-contract, after creating it with
+ *  secp256k1_s2c_commit_context_create the context can be given to a signing
+ *  algorithm via the nonce data argument.
+ *
+ *  This structure is not opaque, but it is strongly discouraged to copy it or read
+ *  or write to it directly. Use the secp256k1_s2c_commit_* instead to access a
+ *  sign-to-contract context.
+ *
+ *  The exact representation of data inside is implementation defined and not
+ *  guaranteed to be portable between different platforms or versions. It is however
+ *  guaranteed to be 128 bytes in size, and can be safely copied/moved.
```

Comment:
one paragraph above, you say that copying is discouraged

Reply:
good catch. I think there's no issue with copying that structure and I think I just copied this sentence from the MuSig session without thinking about it.

Reply:
fixed